### DISCUSSION TITLE
How to get an RSS feed here?

### DISCUSSION THREAD

**Discussion by rouilj** [freeplane/freeplane#344] [Help] [answered]
I followed the sourceforge forum using the RSS feed. Is there something similar for this forum?

**Comment by macmarrum** [comment]
There's an alternative – the `Watch` button (top right) allows to configure `Custom` settings, where `Discussions` can be enabled. The activity is then visible in [`Notifications`](https://github.com/notifications) (the bell icon, top right).

[Notification settings](https://github.com/settings/notifications) allow customizing how the information is delivered (email, web and mobile) and to fine-tune the settings.

I get my notifications via email, where I can see any activity like new posts and replies.

**Comment by rouilj** [comment]


**Reply by macmarrum** [reply]
@rouilj I understand. RSS would definitely be helpful.

Just for the record, for people looking for alternatives to RSS, GitHub Notifications allow to define a filter, e.g. 
* [`repo:freeplane/freeplane is:discussion`](https://github.com/notifications?query=repo%3Afreeplane%2Ffreeplane&query=repo%3Afreeplane%2Ffreeplane+is%3Adiscussion++) to see only Freeplane Discussions
* [`repo:freeplane/freeplane is:discussion is:unread`](https://github.com/notifications?query=&query=repo%3Afreeplane%2Ffreeplane+is%3Adiscussion+is%3Aunread+) – only unread
 
The link can be saved as a bookmark in your browser and viewed quite similar to an RSS feed.

**Comment by dpolivaev** [comment, marked as answer]
Hey everyone, @markcellus has made RSS feeds for Github Discussions available on [openrss.org](https://openrss.org/), a non-profit project I developed to provide RSS feeds for websites that don't have them.

You can get a Discussion RSS feed by navigating to any discussions dashboard page and typing in openrss.org/ right before the github.com part in the browser URL bar.

For example, the RSS feed for the discussions at https://github.com/freeplane/freeplane/discussions are located at https://openrss.org/github.com/freeplane/freeplane/discussions

Works the same for issues and pull requests as well. Let @markcellus know if you run into any issues.

See https://github.com/orgs/community/discussions/31#discussioncomment-3807225

**Reply by markcellus** [reply]
Absolutely, and thanks for the shout @dpolivaev! I can be reached at openrss at protonmail.com or by using the [contact form](https://openrss.org/contact) on the site. :+1:
